tree.go

```diff
@@ -77,6 +77,7 @@ var (
 	text     = newMIME("text/plain", ".txt", magic.Text, html, svg, xml, php, js, lua, perl, python, json, ndJson, rtf, tcl, csv, tsv, vCard, iCalendar, warc)
 	xml      = newMIME("text/xml", ".xml", magic.Xml, rss, atom, x3d, kml, xliff, collada, gml, gpx, tcx, amf, threemf, xfdf, owl2)
 	json     = newMIME("application/json", ".json", magic.Json, geoJson)
+	har      = newMIME("application/json", ".har", magic.HAR)
```

Comment:
Since HAR is a subtype of JSON, you can add it as a child of `json`, like it's done for `geoJson`. The hierarchy will become:
`octet-stream -> text/plain -> json -> har`
instead of 
`octet-stream -> har`.

Reply:
done